### ISSUE TITLE
Not working with Hindi language 

### ISSUE THREAD

**Issue by iamjamilkhan** [daswer123/xtts-webui#66]
I’m getting error while trying hindi language  File "C:\Users\contact\Desktop\xtts-webui-main\venv\lib\site-packages\TTS\tts\models\xtts.py", line 526, in inference
    text = split_sentence(text, language, self.tokenizer.char_limits[language])
KeyError: 'hi'

**Comment by iamjamilkhan** [comment]
No update?

**Comment by bishtyyy** [comment]
i am 

> I’m getting error while trying hindi language File "C:\Users\contact\Desktop\xtts-webui-main\venv\lib\site-packages\TTS\tts\models\xtts.py", line 526, in inference text = split_sentence(text, language, self.tokenizer.char_limits[language]) KeyError: 'hi'

i am also facing the exact same error. All the other languages are working, only hindi is not working.

**Comment by iamjamilkhan** [comment]
Yes all language were working but not hindi and developer not interested to fix it as I’m reported it a long ago

**Comment by shivshankar11** [comment]
bumped

**Comment by daswer123** [comment]
Hello , I recently switched from the official `tts` lib to a fork that is being actively developed - `coqui-tts`, check the ability to work with hindi now, kinda fixed the hindi work there.
